modules/seastate/CMakeLists.txt

```
@@ -39,22 +39,39 @@ add_library(seastlib STATIC
 )
 target_link_libraries(seastlib nwtclibs versioninfolib)
 
+
 # C-bindings interface library
-add_library(seastate_c_binding SHARED 
-  src/SeaState_C_Binding.f90
-)
-target_link_libraries(seastate_c_binding seastlib versioninfolib)
+#   create object instead of directly linking into shared and static -- causes issues in parallel builds
+#   This is only required because we are static linking the library for wavetank
+#   NOTE: target linking at the object, static, and shared libraries.  Different CMake versions handle this
+#         slightly differently with unpredictable results if I don't.
+add_library(seastate_c_binding_object OBJECT src/SeaState_C_Binding.f90)
+target_link_libraries(seastate_c_binding_object seastlib nwtclibs versioninfolib)
```

If you add `PUBLIC` then you shouldn't need to add `target_link_libraries` for the static and shared builds.

```suggestion
target_link_libraries(seastate_c_binding_object PUBLIC seastlib nwtclibs versioninfolib)
```